packages/ocamleditor/ocamleditor.1.13.4~preview.4.10/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+name: "ocamleditor"
+version: "1.1.34~preview.4.10"
+
+synopsis:
+  "OCamlEditor is a GTK+ source code editor and build tool for OCaml"
+description:
+  """It provides many features to facilitate editing code, accessing API reference
+directly from the editor and compiling projects."""
+
+authors: "Francesco Tovagliari <ftovagliari@gmail.com>"
+maintainer: "Vasile Rotaru <vrotaru.md@gmail.com>"
+
+homepage: "https://github.com/vrotaru/ocamleditor"
+bug-reports: "https://github.com/vrotaru/ocamleditor/issues"
+
+build: [["ocaml" "build.ml" "ocamleditor"]]
+install: [["ocaml" "tools/install.ml" "-prefix" prefix]]
+remove: [["ocaml" "tools/uninstall.ml" "-prefix" prefix]]
+
+depends: [
+  "ocaml" {= "4.10.0"}
+  "ocamlfind" {>= "1.4.0"}
```

Comment:
```suggestion
  "ocamlfind" {build & >= "1.4.0"}
```

Is findlib used at runtime or this is just a build dependency?

Reply:
For simple projects with no external dependencies is not used, but if uses the `ocamlfind` at runtime to find the paths/flags to pass to the ocaml compiler while compiling the changed files.

So, short version: it is not just an build dependency.